### DISCUSSION TITLE
is 32 bit build still supported? always fails!

### DISCUSSION THREAD

**Discussion by wbtcpip2** [zhongfly/mpv-build#80] [General]
is 32 bit build still supported? always fails! 

https://github.com/wbtcpip2/mpv-winbuild/actions/runs/6917210198/job/18818147571

**Comment by zhongfly** [comment]
issue like this should report to https://github.com/shinchiro/mpv-winbuild-cmake
Due to the updating of the depends, some times the compilation will fail. This repo is just a helper that use shinchiro/mpv-winbuild-cmake to build mpv.

**Reply by zhongfly** [reply]
https://github.com/shinchiro/mpv-winbuild-cmake/issues/520 
Something wrong with cargo-c or rust-lang
you should wait for shinchiro or rust-lang fixing it.

**Reply by zhongfly** [reply]
update:
add a temporary patch to fix it https://github.com/zhongfly/mpv-winbuild/blob/main/patch/0001-rustup-fix-cargo-c-install.patch
